src/installer/exceptions.py

```diff
@@ -5,5 +5,5 @@ class InstallerError(Exception):
     """All exceptions raised from this package's code."""
 
 
-class InvalidWheelSource(InstallerError):
+class InvalidWheelSource(InstallerError):  # noqa: N818
```

Comment:
Since we are doing a major release, I was thinking that we might do a slight API breakage and rename this to `InvalidWheelSourceError`

Reply:
I don't think this lint rule is worth an API breakage.

If we do rename this, I think it's relatively cheap to add a `InvalidWheelSource = InvalidWheelSourceError`.

Honestly though, I think that name is worse than the current name and there's precedence that exceptions don't need an "error" in their name, like https://docs.python.org/3/library/subprocess.html#subprocess.TimeoutExpired and more in the standard library. "Invalid..." sufficently communicates the situation IMO.

Reply:
None the less, if we want to do this, let's do it in a separate PR.

Reply:
Makes sense. Decided to just ignore the rule. I guess we are smart enough to rule our exception names.